build.sbt

```diff
@@ -1081,7 +1084,7 @@ val createTargetClassesDir = taskKey[Unit]("create target classes dir")
 
 // Don't use these groups for any other projects
 lazy val sparkGroup = project
-  .aggregate(spark, contribs, storage, storageS3DynamoDB, iceberg)
+  .aggregate(spark, contribs, storage, storageS3DynamoDB)
```

Comment:
nit: add back `iceberg` here, right?

Reply:
@lzlfred Can you fix the above comment and try running?